index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <script defer type="module" src="/src/index.tsx"></script>
     <link rel="manifest" href="/manifest.json" />
+    <link rel="icon" type="image/png" href="/icons/icon-72x72.png" />
```

Comment:
we also need the the crazy, non-compliant thing for ios
https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html

Reply:
Sigh. Thanks Apple